wb/device_manager/main.py

```diff
@@ -142,6 +142,30 @@ def __init__(self, failed_ports):
         self.metadata = {"failed_ports": failed_ports}
 
 
+class ReadFWVersionDeviceError(GenericStateError):
+    ID = "com.wb.device_manager.device.read_fw_version_error"
+    MESSAGE = "Failed to read FW version."
+
+
+class ReadFWSignatureDeviceError(GenericStateError):
+    ID = "com.wb.device_manager.device.read_fw_signature_error"
+    MESSAGE = "Failed to read FW signature."
+
+
+class ReadDeviceSignatureDeviceError(GenericStateError):
+    ID = "com.wb.device_manager.device.read_device_signature_error"
+    MESSAGE = "Failed to read device signature."
+
+
+class CompositeDeviceError(GenericStateError):
+    ID = "com.wb.device_manager.device.composite_error"
+    MESSAGE = "Multiple errors occured."
+
+    def __init__(self, error_ids):
+        super().__init__()
+        self.metadata = {"error_ids": error_ids}
```

Comment:
```suggestion
        self.errors = {"error_ids": error_ids}
```

metadata всё-таки другой смысл имеет

Reply:
я вот вообще хочу не composite_error делать, а список ошибок для каждого девайса
но это на фронте менять надо

проблема - мы изначально решили, что error у девайса - должен быть единственным; а тут - вдруг захотели все ошибки (в последнем демо)

=> или composite_error (как у меня сейчас), или список errors (думаю, будет аккуратнее); но надо менять фронт

Reply:
Ну так давай фронт поменяем, там же не сложно

Reply:
сделал со стороны бека (теперь у девайсов не единственная error, а errors [])

Reply:
на фронте добавлю